src/OctoshiftCLI.Tests/gei/Commands/GenerateScriptCommandTests.cs

```diff
@@ -286,6 +286,42 @@ public async Task Invoke_Gets_All_Ado_Repos_For_Each_Team_Projects_When_Ado_Team
             _mockAdoApi.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public async Task Invoke_Returns_For_Ghas_NoSsl_Client_When_NoSsl_Parameter_Is_Provided()
+        {
+
+            // Arrange
+            const string ghesApiUrl = "https://foo.com/api/v3";
+            const string azureStorageConnectionString = "FOO-STORAGE-CONNECTION-STRING";
+
+            _mockGithubApi
+                .Setup(m => m.GetRepos(SOURCE_ORG))
+                .ReturnsAsync(new[] { REPO });
+
+            _mockSourceGithubApiFactory
+                .Setup(m => m.CreateClientNoSsl(ghesApiUrl, It.IsAny<string>()))
+                .Returns(_mockGithubApi.Object);
+
+
+            // Act
+            var args = new GenerateScriptCommandArgs
+            {
+                GithubSourceOrg = SOURCE_ORG,
+                GithubTargetOrg = TARGET_ORG,
+                Output = new FileInfo("unit-test-output"),
+                GhesApiUrl = ghesApiUrl,
+                AzureStorageConnectionString = azureStorageConnectionString,
+                NoSslVerify = true,
+                Sequential = true
+            };
+            await _command.Invoke(args);
+
+            // Assert
+            _script.Should().NotBeEmpty();
+            _mockGithubApi.Verify(m => m.GetRepos(args.GithubSourceOrg), Times.Once);
```

Comment:
IMO it would be also good to explicitly verify that `CreateClientNoSsl` is being called.
```suggestion
            _mockSourceGithubApiFactory.Verify(m => CreateClientNoSsl(ghesApiUrl, It.IsAny<string>()), Times.Once);
            _mockGithubApi.Verify(m => m.GetRepos(args.GithubSourceOrg), Times.Once);
```